kotest-property/src/commonMain/kotlin/io/kotest/property/Gen.kt

```diff
@@ -68,6 +84,11 @@ abstract class Arb<out A> : Gen<A>() {
     */
    abstract fun edgecases(): List<A>
 
+   internal open fun edges(): Option<Edgecase<A>> =
```

Comment:
@sksamuel this is now an internal function; i've decoupled all the edgecases related machineries to the data class. The user-facing api should stay the same.